build_tools/azure/build_and_deploy.yml

```diff
@@ -8,7 +8,16 @@ steps:
   displayName: 'Installing requirements'
 
 - script: python setup.py bdist_wheel
```

Comment:
Can you make this:

```yaml
- script: python setup.py bdist_wheel sdist
```

and it will build both in one go

Reply:
Implementing that, I get this warning I have never seen before:

```bash
/Users/vsts/hostedtoolcache/Python/3.5.7/x64/lib/python3.5/distutils/dist.py:974: UserWarning: 
`build_src` is being run, this may lead to missing
files in your sdist!  You want to use distutils.sdist
instead of the setuptools version:

    from distutils.command.sdist import sdist
    cmdclass={'sdist': sdist}"

See numpy's setup.py or gh-7131 for details.
  cmd_obj.run()
```

Let me look into that before I merge this

Reply:
I think it is because in `setup.py`, if we have `bdist_wheel` in the arguments, we set [`we_be_buildin` to `True`](https://github.com/tgsmith61591/pmdarima/blob/a133de78ba5bd68da9785b061f519ba28cd514cc/setup.py#L99). Therefore, we used [`numpy.distutils.core.setup` instead of `setuptools.setup`](https://github.com/tgsmith61591/pmdarima/blob/a133de78ba5bd68da9785b061f519ba28cd514cc/setup.py#L230-L239). Is that the behavior we want?

Reply:
Gotcha. Let's just leave it as is, then. It's not like it's any significant amount of overhead to run the setup.py twice.